handbook/upgrade/upgrade.md

```diff
@@ -0,0 +1,312 @@
+
+1. Backup your SystemLink Server.
+
+1. Attach a volume to store data captured by `nislmigrate`. This will be referenced as `D:\`.
+
+1. Install `nislmigrate`.
+
+1. Run the command `nislmigrate capture --all --secret <your secret> --dir D:\migration`.
+
+1. Copy the files from the original file storage location to the new file store using one of the following options.
+
+    | Destination | Recommended Tool |
+    | ----------- | ---------------- |
+    | NAS | File Explorer or PowerShell |
+    | SAN | File Explorer or PowerShell |
+    | AWS S3 | [AWS CLI](https://aws.amazon.com/cli/) |
+
+1. Detach `D:\`.
+
+    !!!note
+        Since your original SystemLink Server is still operable, any new data created or consumed by SystemLink Server at this time will not be available to your new server.
+
+1. Provision a new file store for SystemLink.
+
+1. Install and configure the same or a newer version of SystemLink Server.
+
+1. Configure SystemLink to use the newly created file storage location.
+
+1. Install `nislmigrate` on your new SystemLink Server.
+
+1. Attach the `D:\` volume used to capture data from your original SystemLink server.
+
+1. Run the command `nislmigrate restore --all --secret <your secret> --change-file-store-root <new root> --dir D:\migration`.
+
+    !!!note
+        You must use the --change-file-store-root argument flag to update the file meta data the new root location of your file storage. Otherwise, you will not be able to preview or download files. Depending on your configuration, this could be a new drive letter, UNC path, or S3 URL.
+
+        | Migration destination | `nislmigrate` usage |
+        | --------------------- | ------------------- |
+        | New drive and directory | `nislmigrate restore --all --secret <your secret> --change-file-store-root X:\systemlink\data --dir D:\migration` |
+        | UNC path | `nislmigrate restore --all --secret <your secret> --change-file-store-root \\FileShare\systemlink\data --dir D:\migration` |
+        | AWS S3 | `nislmigrate restore --all --secret <your secret> --change-file-store-root s3://yours3bucket/systemlink/data/ --dir D:\migration` |
+
+1. Verify your new SystemLink Server has all the expected migrated data.
+
+### Single Node to Multi Node with PostgreSQL
+
+Complete the following steps to upgrade a single node deployment of SystemLink Server to a multi node deployment where the PostgreSQL instance used by the SystemLink Test Monitor service is hosted on a its own server or replica set.
+
+As of SystemLink 21.3.2 SystemLink supports using PostgreSQL as the database backing the Test Monitor service. This configuration is optional, but provides significant performance improvements. `nislmigrate` does not yet support migrating between PostgreSQL servers or replica sets.
```

Comment:
supports using PostgreSQL as an external database unless you install the side-band package for PostgreSQL that we would have to provide. The next full release will support a local installation of PostgreSQL from the main installer.

Reply:
I included a mention of future support for a single node configuration of Postgres